pkg/resources/envoy/configmap.go

```diff
@@ -36,12 +36,21 @@ import (
 	kafkautils "github.com/banzaicloud/kafka-operator/pkg/util/kafka"
 )
 
-func (r *Reconciler) configMap(log logr.Logger, extListener v1beta1.ExternalListenerConfig) runtime.Object {
+func (r *Reconciler) configMap(log logr.Logger, extListener v1beta1.ExternalListenerConfig,
+	_ v1beta1.IngressConfig, ingressConfigName, defaultIngressConfigName string) runtime.Object {
+	var configMapName string
+	if ingressConfigName == util.IngressConfigGlobalName {
```

Comment:
The logic for determining the value for `configMapName` could be factored out into a separate function.